docs/NSR.md

```diff
@@ -43,7 +43,15 @@ c. the relevant practices implemented by Delegated Third Parties.
 
 **Certificate Management System**: A system used by a CA or Delegated Third Party to process, approve issuance of, or store certificates or certificate status information, including the database, database server, and storage.
 
-**Certificate Systems**: The system used by a CA or Delegated Third Party in providing identity verification, registration and enrollment, certificate approval, issuance, validity status, support, and other PKI-related services.
+**Certificate Systems**: A system used by a CA or Delegated Third Party to access, process, or manage data or provide services related to:
+
+* certificate application;
+* certificate validation;
+* certificate approval;
+* signing;
+* certificate revocation;
+* serving authoritative certificate status information; or
+* key escrow.
```

Comment:
* Make CDN for certificate status use-case explicit as exception
* Update `certificate validation` to describe the action, e.g. `performing validation`
 * DNS lookups are part of validation, so ensure language accounts for that

Reply:
I assume reseller portals that manage requests which are later submitted to the CA's "Certificate Systems", are also excluded.

Reply:
Yes, they should be excluded because the certificate application is the data the CA ultimately receives.
Any pre-checks done by a reseller portal should be out of scope because they may not have an influence on the outcome of the CA's validation procedures. All relevant validation has to be performed on the CA's side of the process.